cmd/prana/main.go

```diff
@@ -15,6 +16,7 @@ var CLI struct {
 	Shell       commands.ShellCommand       `cmd:"" help:"Start a SQL shell for Prana"`
 	UploadProto commands.UploadProtoCommand `cmd:"" help:"Upload a protobuf file descriptor set that can be used by Prana to decode sources"`
 	Addr        string                      `help:"Address of PranaDB server to connect to." default:"127.0.0.1:6584"`
+	TLSConfig   tls.TLSConfig               `help:"TLS client configuration" embed:"" prefix:""`
```

Comment:
Can you show a usage example of how the CLI would be configured at the command line with TLS? It's not clear to me by looking at this.

Reply:
Explained above. The API TLS can use CLI arguments prefixed with `api-tls-`: `--api-tls-ca-cert`, `--api-tls-key`, etc. 
I'm planning on using different prefix for Raft TLS: `raft-tls-ca-cert`, `raft-tls-key`, etc.

Reply:
raft tls?
I don't think there is a requirement to encrypt raft tls at this point